### PR TITLE
install.sh: set LD_LIBRARY_PATH during install

### PR DESCRIPTION
**Pull request by satmandu**
Revert to the working LD_LIBRARY_PATH setting code from this summer...

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -292,12 +292,13 @@ ln -sfv "../lib/crew/bin/crew" "${CREW_PREFIX}/bin/"
 
 echo_out "Set up and synchronize local package repo..."
 
-# Set LD_LIBRARY_PATH so crew doesn't break on i686 and
-# the mandb postinstall doesn't fail.
+# Set LD_LIBRARY_PATH so crew doesn't break on i686, xz doesn't fail on
+# x86_64, and the mandb postinstall doesn't fail.
 if [[ "$ARCH" == "i686" ]]; then
   echo "LD_LIBRARY_PATH=$CREW_PREFIX/lib:/lib" >> "$CREW_PREFIX"/etc/env.d/00-library
-  export LD_LIBRARY_PATH=$CREW_PREFIX/lib:/lib
```
Comment (uberhacker) [inline install.sh:299]: Wouldn't we still need this for i686?

Comment (satmandu) [reply install.sh:299]: Hmmm.... You're right...

Comment (satmandu) [reply install.sh:299]: Checking if it is needed. If it is, we can change the next line to: 
```
export LD_LIBRARY_PATH="${CREW_PREFIX}/lib${LIB_SUFFIX}:/lib${LIB_SUFFIX}"
```

Comment (satmandu) [reply install.sh:299]: Wasn't needed. i686 install went fine.

**Review by uberhacker — APPROVED** [review]

**Pull request merged**